Arelle/Arelle
# remove pre-REC TLB references, adapt to conformant structural model

#### Reason for change
Table Linkbase work to prepare for Certification in 2022

#### Description of change
Remove support for pre-REC versions of table linkbase (Eurofiling and pre-REC draft versions), add PWD 1.1 support.
Adapt to conformant structural model.
Update table linkbase implementation to pass certification suite.

#### Steps to Test
Use certification suite with layout model comparisons (built-in), examine GUI and html outputs (manual).
Ensure test script results (script operation compares expected and actual Layout Model outputs) and review GUI outputs on screen and html outputs produced against expected html outputs.

Needs regression testing capability for GUI, can something like Cypress be used?

**review**:
@Arelle/arelle
